lib/mintel/dashboards/services/omni/analytics.libsonnet

```diff
@@ -98,4 +98,49 @@ local promQuery = import 'components/prom_query.libsonnet';
       )
     ),
  ]),
+
+  dashboardRequest(span=12)::
```

Comment:
Same for those ... i think we should name them with something other than `dashboard` or they will look like we are talking about grafana dashboards

Reply:
fair, I have updated them